tests/python/test_random.py

```diff
@@ -2,11 +2,7 @@
 from taichi import approx
 
 
-def archs_support_random(func):
```

Comment:
I have grep-ed this function in the project and it seems this function is not used anywhere, so I delete it in this PR. Please feel free to correct me if I am missing something.